src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/components/HmrcReportTechnicalIssue.scala.html

```diff
@@ -22,7 +22,7 @@
   <a lang="@params.language.code"
     hreflang="@params.language.code"
     class="@toClasses("govuk-link hmrc-report-technical-issue ", classes.getOrElse(""))"
-    @if(params.referrerUrl.nonEmpty) { rel="noreferrer noopener" }
+    rel="noreferrer noopener"
```

Comment:
The twirl template was not aligned with the hmrcfrontend nunjucks template. In the nunjucks template, the rel attribute is not optional.